reenact/templates/reenact/tabs/scenarios.html

```diff
@@ -4,15 +4,17 @@ <h2 class="text-lg font-bold mb-4">Wählen Sie ein Szenario aus</h2>
     <div id="scenarioTabs" class="flex flex-col gap-4">
       {{ scenarios|json_script:"scenarios-data" }}
       {% for scenario in scenarios %}
-        <button class="scenario-button {% if forloop.first %}selected{% endif %}"
+        <button class="scenario-button"
                 onclick="updateScenario('{{ scenario.title }}', '{{ scenario.description }}', this); reload_main_chart('{{ scenario.number }}');">
           {{ scenario.number }}: {{ scenario.name }}
         </button>
       {% endfor %}
     </div>
   </div>
-  <div class="p-4">
-    <h2 id="scenarioTitle" class="text-lg font-bold mb-2">{{ scenarios.0.title }}</h2>
-    <p id="scenarioDescription" class="text-gray-700 mb-4">{{ scenarios.0.description }}</p>
-  </div>
+  <div id="scenario_box" class="p-4"></div>
 </div>
+
+  <template id="scenario_info">
+    <h2 id="scenarioTitle" class="text-lg font-bold mb-2"></h2>
+    <p id="scenarioDescription" class="text-gray-700 mb-4"></p>
+  </template>
```

Comment:
Ah sorry - this is not what i meant.
I wanted to integrate multiple templates, one for each scenario, which are then clonend and loaded into related div.
I.e.
```
<template id="scenario_wind">
    <h2 class="text-lg font-bold mb-2">Wind </h2>
    <p class="text-gray-700 mb-4">lorem ipsum</p>
  </template>
<template id="scenario_wind_pv">
    <h2 class="text-lg font-bold mb-2">Wind & PV</h2>
    <p class="text-gray-700 mb-4">lorem ipsum <img> and other HTML stuff which could be integrated here</p>
  </template>
  ```
Then template must be identified by related id and loaded in function. Function would not get title and description as parameters, only scenario name or ID...